src/examples/3d_fluid_sim/fluid_sim.hpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#ifndef DIM
+#define DIM 3
+#endif
+
+#if DIM == 2
+#define DIM 3
+#endif
+
+#include <random>
+
+#include "../../wash/wash.hpp"
+
+#define PI 3.1415926
+
+wash::Vec3D randomSpherePoint() {
+    std::uniform_real_distribution<double> unif(0.0, 1.0);
+    std::default_random_engine re;
+
+    wash::Vec3D pos{unif(re), unif(re), unif(re)};
+    auto mag = pos.magnitude();
+
+    if (mag == 0.0) {
+        return wash::Vec3D{0.0, 0.0, 0.0};
```

Comment:
Should be able to write `wash::Vec3D{}` as default initialiser is all zeroes

Reply:
Changed.